### PR TITLE
Update config. Cleanup

### PR DESCRIPTION
**Pull request by merqlove**
PR only for mix.exs

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -1,36 +1,46 @@
...
+      homepage_url: @project_url,
       description: "State machine pattern for Ecto. I tried to make it similar as possible to ruby's gem 'aasm'",
       package: package
    ]
   end
 
-  defp elixirc_paths(:prod), do: ["lib"]
-  defp elixirc_paths(:dev),  do: ["lib", "test/dummy/web", "test/dummy/lib"]
-  defp elixirc_paths(:test), do: ["lib", "test"]
+  defp elixirc_paths(:test), do: elixirc_paths ++ ["test/support", "test/dummy"]
```
Comment (asiniy) [inline mix.exs:23]: Make a tabularization, please

File: mix.lock [hunk 2]
```diff
@@ -1,8 +1,8 @@
-%{"connection": {:hex, :connection, "1.0.2"},
-  "db_connection": {:hex, :db_connection, "0.2.5"},
-  "decimal": {:hex, :decimal, "1.1.2"},
-  "ecto": {:hex, :ecto, "2.0.0-rc.5"},
-  "ex_machina": {:git, "https://github.com/thoughtbot/ex_machina.git", "84ccf518a1b025f3ff855aa743defc20d5e91b4e", []},
-  "ex_spec": {:hex, :ex_spec, "1.0.0"},
-  "poolboy": {:hex, :poolboy, "1.5.1"},
-  "postgrex": {:hex, :postgrex, "0.11.1"}}
+%{"connection": {:hex, :connection, "1.0.3", "3145f7416be3df248a4935f24e3221dc467c1e3a158d62015b35bd54da365786", [:mix], []},
```
Comment (asiniy) [inline mix.lock]: Please checkout these changes, they are unnecessary

File: mix.exs (hunk 1 above)
Comment (merqlove) [inline mix.exs:23]: What do you mean?

File: mix.lock (hunk 2 above)
Comment (merqlove) [inline mix.lock]: Ok

File: mix.exs (hunk 1 above)
Comment (merqlove) [inline mix.exs:23]: as you can see ["test/support", "test/dummy"] will be ["test/support"] in next PRs

Comment (asiniy) [inline mix.exs:23]: I explained the essential of tabularization in another PR. If

> ["test/support", "test/dummy"] will be ["test/support"] in next PRs

let's move this kind of change to the next PR

File: test/test_helper.exs [hunk 3]
```diff
@@ -1,5 +1,3 @@
 ExUnit.start()
 
 { :ok, _ } = Dummy.Repo.start_link
-{ :ok, _ } = Application.ensure_all_started(:postgrex)
```
Comment (asiniy) [inline test/test_helper.exs:4]: This is advice of these packages maintainers. I prefer to store them as is until according changes go out of docs.

**Comment by asiniy** [comment]
I think it's cool enough and ready to be merged except things I mentioned above. Let's `amend` them and merge!

File: test/test_helper.exs (hunk 3 above)
Comment (merqlove) [inline test/test_helper.exs:4]: They are in mix.exs already:

``` elixir
def app_list(:test), do: app_list ++ [:ecto, :postgrex, :ex_machina]
```

Just a replacement.

**Comment by merqlove** [comment]
Yeah, i have rebased & pushed this commit.

**Comment by asiniy** [comment]
Thanks! :+1:

**Pull request merged**